python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -35,13 +35,22 @@ def get_evaluator(config):
     """ Get evaluator class
     """
     if config.task_type == "node_classification":
-        return GSgnnAccEvaluator(config.eval_frequency,
-                                 config.eval_metric,
-                                 config.multilabel,
-                                 config.use_early_stop,
-                                 config.early_stop_burnin_rounds,
-                                 config.early_stop_rounds,
-                                 config.early_stop_strategy)
+        if isinstance(config.multilabel, dict):
+            return GSgnnAccEvaluator(config.eval_frequency,
+                                    config.eval_metric,
+                                    config.multilabel[config.eval_target_ntype],
+                                    config.use_early_stop,
+                                    config.early_stop_burnin_rounds,
+                                    config.early_stop_rounds,
+                                    config.early_stop_strategy)
+        else:
+            return GSgnnAccEvaluator(config.eval_frequency,
+                                    config.eval_metric,
+                                    config.multilabel,
+                                    config.use_early_stop,
+                                    config.early_stop_burnin_rounds,
+                                    config.early_stop_rounds,
+                                    config.early_stop_strategy)
```

Comment:
the only difference is `multilabel`? if so, can we write it like this:
```
multilabel = config.multilabel[config.eval_target_ntype] if isinstance(config.multilabel, dict) else config.multilabel
```

Reply:
solved